options/ansi/generic/stdio-stubs.cpp

```diff
@@ -898,6 +898,10 @@ int getchar(void) {
 	return fgetc(stdin);
 }
 
+char* gets(char* s){
```

Comment:
maybe we should get not expose `gets` by default (and name its symbol something else, or this a static inline function, to prevent an undeclared gets linking properly)? it is actively being removed from standards:
```
       LSB  deprecates  gets().   POSIX.1-2008 marks gets() obsolescent.  ISO C11 removes the specification of gets() from the C language, and since
       version 2.16, glibc header files don't expose the function declaration if the _ISOC11_SOURCE feature test macro is defined.
```
from glibc `gets(3)`

Reply:
Maybe this is more something for when we actually start adhering to feature test macros?